SingularityUI/app/actions/log.es6

```diff
@@ -256,6 +256,7 @@ export const taskGroupFetchPrevious = taskGroupId =>
     }
 
     dispatch({taskGroupId, type: 'LOG_REQUEST_START'});
+    tasks = _.without(tasks, undefined);
```

Comment:
I'm not sure, there seems to be an issue with how the actions are removing instances from the list but I wasn't able to pinpoint where. This might actually be a no op since this is related to the same problem on QA and PROD.